swagger/swagger-generator/generator-core/src/test/java/org/apache/servicecomb/swagger/generator/core/TestSwaggerGenerator.java

```diff
@@ -26,6 +26,7 @@
 import org.junit.Assert;
```

Comment:
remove the unused improt is better, you can find this in several modified files, so you can double check

Reply:
@david6969xin Great. We can enable the unused import check on checkstyle. Can you work on this?

Reply:
I will fix it.